test/fw/ptl.sh

```diff
@@ -71,6 +71,8 @@ else
 			python_dir=$( /bin/ls -1 ${PTL_PREFIX}/lib )/site-packages
 			[ -d "${PTL_PREFIX}/bin" ] && export PATH="${PATH}:${PTL_PREFIX}/bin"
 			[ -d "${PTL_PREFIX}/lib/${python_dir}" ] && export PYTHONPATH="${PYTHONPATH}:${PTL_PREFIX}/lib/${python_dir}"
+			[ -d "${__PBS_EXEC}/lib/python/altair" ] && export PYTHONPATH="${PYTHONPATH}:${__PBS_EXEC}/lib/python/altair"
```

Comment:
with only this change it will force PTL to go in PTL_API mode. we need to remove [this](https://github.com/openpbs/openpbs/blob/master/test/fw/ptl/lib/pbs_testlib.py#L4762) if condition so by default PTL go in PTL_CLI mode (which is current default)

Reply:
As I understand Python modules, this should not be an issue.  The PTL modules explicitly import `ptl.lib.pbs_ifl`.  The Python import mechanism shouldn't ever match the module in the PBS_EXEC directory space given that the ptl/lib directory does not exist in that space.  Did I overlook something?  Granted I am making the assumption that ptl/lib doesn't exist in path I added; and while presently true, I'm happy to address the issue further if you have suggestions on further ways to mitigate a possible inappropriate match and import.

Reply:
@toonen You are right, sorry I overlooked, it won't cause any problem as we don't have ptl/lib in PBS_EXEC/lib/python/altair. No need of any changes for this.